--- /etc/xdg/foot/foot.ini	2023-10-24 00:48:04.000000000 +0900
+++ .config/foot/foot.ini	2023-12-20 17:48:50.760717188 +0900
@@ -8,7 +8,7 @@
 # title=foot
 # locked-title=no
 
-# font=monospace:size=8
+font=Iosevka Nerd Font Mono:size=16
 # font-bold=<bold variant of regular font>
 # font-italic=<italic variant of regular font>
 # font-bold-italic=<bold+italic variant of regular font>
@@ -20,7 +20,7 @@
 # underline-offset=<font metrics>
 # underline-thickness=<font underline thickness>
 # box-drawings-uses-font-glyphs=no
-# dpi-aware=no
+dpi-aware=no
 
 # initial-window-size-pixels=700x500  # Or,
 # initial-window-size-chars=<COLSxROWS>
@@ -223,3 +223,6 @@
 # select-row=BTN_LEFT-4
 
 # vim: ft=dosini
+
+[main]
+include=/usr/share/foot/themes/tempus-classic
